assets/css/settings.coffee

```diff
@@ -0,0 +1,20 @@
+class Settings
+  default_settings:
+    theme: 'default-theme'
+  constructor: (mainDiv, data) ->
+    @mainDiv = mainDiv
+    @data = data
+  bind: () ->
+    settings = this
+    @mainDiv.find("#theme-selection").on 'input', (e) ->
+      theme = @value
+      settings.data.setSetting('theme', theme)
+      settings.changeStyle(theme)
+  loadRenderSettings: () ->
+    theme = @getSetting 'theme'
+    @changeStyle(theme)
+    @mainDiv.find("#theme-selection").val theme
+  getSetting: (setting) -> # TODO: make data.coffee populate default settings directly during setup instead
```

Comment:
Jeff, thoughts on how to handle making the datastore load process use a 'default settings' object? The current way certainly works but I'd rather just write default settings to the datastore if they're not there yet.